evap/student/templates/student_vote.html

```diff
@@ -132,6 +132,28 @@ <h3 class="mb-3">{{ course.name }} ({{ course.semester.name }})</h3>
             // (the data gets deleted every time the form is submitted, i.e. also when the form had errors and is displayed again)
             sisyphus.saveAllData();
 
+            // Show a warning on textareas if they only contain meaningless text (punctuation, k.A., n/a, ...)
+            function isTextareaInvalid(textarea) {
+                return textarea.val().length > 0 && ["", "ka", "na"].includes(textarea.val().replace(/\W/g,'').toLowerCase());
+            }
+            function updateTextareaWarning(textarea) {
+                let showWarning = isTextareaInvalid(textarea);
+                textarea.parents('tr').find('.warning-label').toggleClass('d-none', !showWarning);
+                textarea.toggleClass('warning-border', showWarning);
+            }
+            var warningdelayTimer;
+            $('textarea').on('input', function() { // Run the check on input
+                let textarea = $(this);
+                clearTimeout(warningdelayTimer);
+                warningdelayTimer = setTimeout(() => updateTextareaWarning(textarea), 300);
```

Comment:
inline `textarea`?

Reply:
Oh I originally thought that wouldn't work, because it did not with an anonymous function, but it does work with lambda expressions :+1: